seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -394,18 +435,64 @@
       "translations": {}
     },
     {
+      "id": "587d7dac367417b2b25184d3",
+      "title": "Basic Data Structures: Introduction to Objects",
```

Comment:
Similar to above: I think this can be "Introduction to Objects". Remove "Basic Data Structures:"